aggregations/csv/near_ecosystem_entities.csv

```diff
@@ -1,149 +1,153 @@
-skyward-finance,Skyward Finance,Enable fair token distribution and price discovery for projects built on NEAR Protocol.,https://skyward.finance/,"defi, app, guild",launched,skyward.near
-solacefi,Solace.Fi,Solace is a decentralized protocol that provides coverage for liquidity providers in DeFi.,https://solace.fi/,defi,,
-sputnik-dao,SputnikDAO,Launch pad for your next DAO. Make decisions and manage funds with your community.,https://sputnik.fund,dao,,sputnikdao.near
-stake-gg,Stake.gg,The Prediction Market Platform Built By Gamers for Gamers,https://www.stake.gg/,"app, gaming, defi",building,
-staked,Staked,Staking nodes for institutional investors.,https://staked.us/,"infrastructure, validator",launched,
-state-of-the-dapps,State of the Dapps,Explore decentralized applications.,https://www.stateofthedapps.com/,"tools, analytics",,
-sushiswap,SushiSwap,"Swap, earn, stack yields, lend, borrow, leverage all on one decentralized, community driven platform.",https://sushi.com/,"defi, amm, app",,
-swaggerdao,SwaggerDAO,"SWAGGER is an international network of designers, artists, creatives and developers summoned around the common purpose of bringing fashion NFTs to the mainstream.",https://swaggerdao.com/,guild,,
-swine-guild,06-01-2020,Swine Guild,Making memes on demand and for the NEAR Ecosystem :),guild,,
-titi-protocol,TiTi Protocol,Multi-asset Reserve Stablecoin based on decentralized Monopoly-AMM,http://titi.finance/,"app, defi",,
-transak,Transak,Global fiat on/off-ramp for apps and wallets.,https://transak.com/,tools,,
-transform-infinity,Transform Infinity,A Creative Russian-Ukrainian-English speaking Guild,,guild,,
-trust-wallet,Trust Wallet,Fast and secure Ethereum wallet that’s easy to use.,https://trustwallet.com/,"infrastructure, wallet",launched,
-Verida,Verida,A network of personal data owned and controlled by users.,https://www.verida.io/,"infrastructure, tools, wallet, guild",building,
-vital-point-guild,Vital Point Guild,"A guild focussed on tackling governance, decision support, and data integrity challenges using blockchain, mixed reality, and data science.",https://vitalpoint.ai/,"guild, education",,
-vorto-gaming,Vorto Gaming,Blockchain gaming company from Sweden.,https://vortogaming.com/,gaming,,
-wabbajack-near-design,Wabbajack NEAR Design,The official communication channel for the NEAR Protocol design/art/meme guild known as Wabbajack.,,guild,,
-walletconnect,WalletConnect,The Digital Collectible Hub of Esports,https://walletconnect.org/,wallet,,
-wizards-guild,Wizards Guild,A wizarding school for the dark arts of DeFi.,,"defi, guild",,
-zed-run,ZED Run,Build an empire with your own champion racehorses.,https://www.zed.run/,"gaming, app",launched,
-zen-bikers-guild,Zen Bikers Guild,A thematic guild with the reharmonisation of creativity with technology.,https://gov.near.org/t/zen-bikers-guild/4221,guild,,
+Slug,Title,Oneliner,Website,Category,Status,Contract,Logo

```

Comment:
FYI, given the insert statement that you defined:

https://github.com/near/near-analytics/blob/0dfadc2cbc5cc0cabaab97fd971af469c74ed46e/aggregations/db_tables/near_ecosystem_entities.py#L31-L36

It won't update the records